File: app/src/main/java/com/dominikgaller/alpinebooking/booking/inbound/rest/ConfirmTourBookingResponse.java
package com.dominikgaller.alpinebooking.booking.inbound.rest;

/**
 * Response DTO returned by the UC02 – ConfirmTourBooking endpoint.
 *
 * <p>SDD: See {@code documentation/use-cases/uc02-confirm-tour-booking.spec.md}, section 3.
 */
public record ConfirmTourBookingResponse(String status) {
}

File: src/main/kotlin/com/example/contractmanagement/booking/inbound/rest/response/ConfirmTourBookingResponse.kt
package com.example.contractmanagement.booking.inbound.rest.response

/**
 * REST response body for UC02 — ConfirmTourBooking.
 *
 * SDD: see `documentation/use-cases/uc02-confirm-tour-booking.spec.md` § 9.
 */
data class ConfirmTourBookingResponse(
    val status: String,
)
